### DISCUSSION TITLE
Adobe Cirrus and RTMFP server, what are the available replacements?

### DISCUSSION THREAD

**Discussion by skramovskiy** [airsdk/Adobe-Runtime-Support#1634] [General]
Good afternoon
Tell me, Adobe Cirrus is unavailable - how to replace it? 
My task is simple - the exchange of text information?

**Comment by Fancy2209** [comment]
> Good afternoon Tell me, Adobe Cirrus is unavailable - how to replace it? My task is simple - the exchange of text information?

The only replacement I know is hosting your own RTMFP Server using https://github.com/MonaSolutions/MonaServer2
